deploy/kubernetes/examples/demo-volumereplication.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: replication.storage.openshift.io/v1alpha1
+kind: VolumeReplication
+metadata:
+  name: demo-volumereplication
+  namespace: default
+spec:
+  volumeReplicationClass: demo-volumereplicationclass
+  replicationState: primary
+  dataSource:
+    kind: PersistentVolumeClaim
+    name: demo-pvc-file-system  # Should be in same namespace as VolumeReplication
```

Comment:
missing EOL

Reply:
"# Place in the same namespace as Volume Replication"
If "place" isn't the right word, please suggest an alternative. We should avoid using "should"

Reply:
@oriyarde "place"?

Reply:
this was copied from [here](https://github.com/csi-addons/volume-replication-operator/blob/faf6e90cec37092f153007e667a997d00d0cd76c/README.md#volumereplication)
"ensure"?

Reply:
We can write "Ensure that this is in the same namespace as VolumeReplication."

Reply:
done

Reply:
@rivkap could I resolve this?

Reply:
LGTM. Thanks